app/modules/tickets/delete_ticket/delete_ticket.controller.js

```diff
@@ -0,0 +1,28 @@
+const {validationResult} = require ('express-validator');
+const {deleteTicketQuery,findTicket} = require('./delete_ticket.query');
+const {authorize} = require('../../../middlewares/auth');
+// const {sequelize} = require('../../../../models/index');
+
+const deleteTicket = async(req,res)=>{
+console.log('dlt ticket hit');
+const validation = validationResult(req);
+
+if(!validation.isEmpty())
+return res.send(validation);
+
+await authorize (req,res);
+console.log('req user object',req.user.id);
+
+const ticket = await findTicket(req.params.id);
+console.log('user ticket',ticket.CreatedBy_Id);
+
+if(req.user.id !== ticket.CreatedBy_Id)
+res.send('invalid user');
+
+const dltTicket = deleteTicketQuery(req.params.id);
```

Comment:
don't use short names.
Naming should be in such a manner that anyone can understand the use of the function.
Even if it's a bit lengthy. It doesn't matter.